src/player_display.cpp

```diff
@@ -349,15 +347,15 @@ static void draw_proficiencies_tab( const catacurses::window &win, const unsigne
             name = trim_by_length( cur.id->name(), width );
         }
         const nc_color col = focused && i == line ? hilite( cur.color ) : cur.color;
-        y += fold_and_print( win, point( 0, y ), width, col, name );
+        fold_and_print( win, point( 0, 1 + i - range.first ), width, col, name );
```

Comment:
`y += ` is used here because `fold_and_print` can print multiple lines when the text is long. If this does happen I recommend caching the folded strings and print the strings and the scroll bar according to the number of lines folded. Alternatively you can use `trim_and_print` here like the other tabs.

Reply:
Actually, it was already trimmed in the previous if-else statement, so it always returns 1. Maybe using it is therefore overkill?

Reply:
`print_colored_text` should be enough then.